draft-ietf-quic-transport.md

```diff
@@ -2670,9 +2670,10 @@ complete.  Though the values of some fields in the packet header might be
 redundant, no fields are omitted.  The receiver of coalesced QUIC packets MUST
 individually process each QUIC packet and separately acknowledge them, as if
 they were received as the payload of different UDP datagrams.  For example, if
-decryption fails (because the keys are not available or any other reason), the
-the receiver MAY either discard or buffer the packet for later processing and
-MUST attempt to process the remaining packets.
+decryption fails (because the keys are not available, the UDP datagram is a PMTU
+probe (see {{pmtu-probes-src-cid}}), or any other reason), the the receiver MAY
```

Comment:
I'm not sure this needs to be called out separately.  If Handshake keys have been dropped, the keys won't be available and you've already covered this.

Reply:
It looks technically redundant, but it works to point out this use case in the _Coalescing Packets_ section. The other alternative would be to mention PMTU probes earlier (in the in the second paragraph, for example):

```suggestion
Using the Length field, a sender can coalesce multiple QUIC packets into one UDP datagram.
This can reduce the number of UDP datagrams needed to complete the cryptographic
handshake and start sending data.  This can also be used to construct PMTU probes (see
{{pmtu-probes-src-cid}}).  Receivers MUST be able to probe (see {{pmtu-probes-src-cid}}),
or any other reason), the the receiver MAY process coalesced packets.
```

I do wonder if mentioning this too early would distract from the main purpose of coalesced packets -- reducing the number of UDP datagrams during handshake.

Reply:
@MikeBishop, I am trying to make sure there are no unanswered concerns left about this text.

I know you've approved the text they way it is.  So you you think it is ok as is (and we can Resolve this), or should we work on making a change similar to the one I've outlined above?  Or something else?

Reply:
The idea behind your suggested change reads better, yes.  The specific suggested change has other issues that I assume are caused in part by other lines having changed:

> Receivers MUST be able to probe (...), or any other reason), the the receiver MAY process coalesced packets.

Reply:
The new commit updates the text as requested / discussed.  Resolving this.